### PR TITLE
Micos implementation

### PR DESCRIPTION
**Pull request by edobN**

### CODE REVIEW

File: esrfconcert/devices/motors/micos.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # TODO: do we still need this?
```
Comment (tfarago) [inline esrfconcert/devices/motors/micos.py:1]: @edobN, here I poke myself @tfarago to remind me to remove this comment before merging.

File: esrfconcert/devices/motors/micos.py [hunk 2]
```diff
@@ -83,42 +84,42 @@ def __init__(self, controller, index, host, port):
         super(LinearMotor, self).__init__()
         _Base.__init__(self, controller, index, host, port)
 
-    def _get_position(self):
-        return self._get_position_in_steps() * q.mm
+    async def _get_position(self):
+        return await self._get_position_in_steps() * q.mm
```
Comment (edobN) [inline esrfconcert/devices/motors/micos.py:87]: This I was not sure about @tfarago 
I saw lines that looked rather like
return (await self._device['position']).value * self['position'].unit
anyway I figured I don't need to access the .value property here, so I don't need (...)

Comment (tfarago) [reply esrfconcert/devices/motors/micos.py:87]: Yes, this will work.

**Comment by tfarago** [comment]
@edobN, I had to do some cleanup. Please delete your local branch and do "git checkout -t origin/micos-implementation". That will get you back on track. Then, please install flake8, "pip install --user flake8", and then do "flake8 esrfconcert/devices/" when in the repo directory, and then all files which this PR touches must have zero output from flake8, meaning the code is clean.

File: esrfconcert/tests/test_micos_motors.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+MICOS_HOST = '160.103.39.110'
+MICOS_PORT = 6542
+
+
+class TestLinearMotor(TestCase):
+
+    """Simple sanity tests."""
+
+    def setUp(self):
+        self.motor = LinearMotor('foo', 1234, MICOS_HOST, MICOS_PORT)
```
Comment (tfarago) [inline esrfconcert/tests/test_micos_motors.py:20]: Here we will need a valid controller and axis index.

**Comment by MarcusZuber** [comment]
If your implementation works really 100%, I can move all this into tango. If you check all the system behaviour with this implementation it will be only very little work to migrate it.

**Comment by tfarago** [comment]
Thanks for the offer, we are currently checking if by chance it's not already being ported into bliss.

**Pull request merged**